_config.yml

```diff
@@ -1,2 +1,31 @@
-title: Adfinis Helm Charts
-theme: jekyll-theme-primer
+locale                  : "en-US"
+title                   : adfinis/helm-charts
```

Comment:
why diff title?

Reply:
Do we really want this:
![image](https://user-images.githubusercontent.com/51193103/219592130-90a540fe-632c-4568-9069-cb16c4adca6b.png)
instead of
![image](https://user-images.githubusercontent.com/51193103/219592009-60d0cc1e-3e67-4408-9f2c-701efe31c280.png)